modules/reference/pages/broker-properties.adoc

```diff
@@ -0,0 +1,873 @@
+
+== Broker
+
+=== admin
+
+IP address and port of the admin server.
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Default:* `{model::broker_endpoint(net::unresolved_address("127.0.0.1", 9644))}`
+
+---
+
+=== admin_api_doc_dir
+
+Path to the admin API documentation directory.
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* string
+
+*Default:* `/usr/share/redpanda/admin-api-doc`
+
+---
+
+=== admin_api_tls
+
+TLS configuration for the Admin API.
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Default:* `null`
+
+---
+
+=== crash_loop_limit
+
+A limit on the number of consecutive times a broker can crash within one hour before its crash-tracking logic is reset. This limit prevents a broker from getting stuck in an infinite cycle of crashes.
+
+If `null`, the property is disabled and no limit is applied.
+
+The crash-tracking logic is reset (to zero consecutive crashes) by any of the following conditions:
+
+* The broker shuts down cleanly.
+* One hour passes since the last crash.
+* The broker configuration file, `redpanda.yaml`, is updated.
+* The `startup_log` file in the broker's <<data_directory,data_directory>> is manually deleted.
+
+*Units*: number of consecutive crashes of a broker.
+
+*Requires Restart:* Yes
+
+*Nullable:* Yes
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Accepted values:* [`0`, `4294967295`]
+
+*Default:* `5`
+
+---
+
+=== data_directory
+
+Path to the directory for storing Redpanda's streaming data files.
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* string
+
+*Default:* `null`
+
+---
+
+=== developer_mode
+
+CAUTION: Enabling `developer_mode` isn't recommended for production use.
+
+Flag to enable developer mode, which skips most of the checks performed at startup.
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== emergency_disable_data_transforms
+
+Override the cluster enablement setting and disable WebAssembly powered data transforms. Only used as an emergency shutoff button.
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== empty_seed_starts_cluster
+
+Controls how a new cluster is formed. This property must have the same value in all brokers in a cluster.
+
+If `true`, an empty <<seed_servers,seed_servers>> list denotes that this broker should form a cluster. At most, one broker in the cluster should be configured with an empty seed_servers list. If no such configured broker exists, or if configured to be `false`, then all brokers denoted by the seed_servers list must be identical in their configurations, and those brokers form the initial cluster.
+
+TIP: This is set to `true` by default for backward compatibility, but Redpanda recommends setting `empty_seed_starts_cluster` to `false`.
```

Comment:
```suggestion
TIP: This is set to `true` by default for backward compatibility, but Redpanda recommends setting `empty_seed_starts_cluster` to `false`.

If `true`, an empty <<seed_servers,seed_servers>> list denotes that this broker should form a cluster. At most, one broker in the cluster should be configured with an empty seed_servers list. If no such configured broker exists, or if configured to be `false`, then all brokers denoted by the seed_servers list must be identical in their configurations, and those brokers form the initial cluster.
```
ENG: Do we still want to include this "If `true`" part? Or should that be deleted, or point to doc: https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/manual/production/production-deployment/#configure-the-seed-servers?